tests/tests_pytorch/utilities/test_rank_zero.py

```diff
@@ -61,12 +62,3 @@ def test_rank_zero_priority(environ, expected_rank):
     """Test the priority in which the rank gets determined when multiple environment variables are available."""
     with mock.patch.dict(os.environ, environ):
         assert _get_rank() == expected_rank
-
-
-@pytest.mark.parametrize("env_vars", [{"RANK": "0"}, {"RANK": "1"}, {"RANK": "4"}])
-def test_rank_prefixed_message_with_env_vars(env_vars):
```

Comment:
I can't see where this moved. Is it in lightning-utilities?

Reply:
Yes

Reply:
https://github.com/Lightning-AI/utilities/blob/c27592abab2b24578753794c9bf005bc7f627595/tests/unittests/core/test_rank_zero.py#L12-L18